### DISCUSSION TITLE
当我读李娟时我在想些什么 — 码志

### DISCUSSION THREAD

**Discussion by giscus** [mzlogin/blog-comments#276] [Announcements]
# 当我读李娟时我在想些什么 — 码志

最近李娟的作品很火，我先是在前同事的微博里看到了《遥远的向日葵地》，然后又从现同事的朋友圈里知道了《我的阿勒泰》。

https://mazhuang.org/2024/06/07/read-lijuan-think-myself/

**Comment by Quanzhitong** [comment]
感谢分享，稀罕这句话：“我企图从她的文字里窥探到写作的奥义，但最终只看见了自己内心的贫瘠”。
